cli/bench/console.js

```diff
@@ -1,8 +1,3 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 const count = 100000;
-
-const start = Date.now();
 for (let i = 0; i < count; i++) console.log("Hello World");
-const elapsed = Date.now() - start;
-const rate = Math.floor(count / (elapsed / 1000));
-console.log(`time ${elapsed} ms rate ${rate}`);
```

Comment:
just using time? why this change?

Reply:
Removing extra stuff so it can easily measured using `time` and `hyperfine`.